<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFile</class>
 <widget class="QDialog" name="NewFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建仿真</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_unit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_mum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>μm</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mm">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>空间单位：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>使用算法：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_backend">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADI-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CN-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SS-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOD-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WCS-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HIE-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CDLT-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SO-FDTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PITD-FDTD</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文件名：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>.json</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>48</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>（默认位于emulation目录下）</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>时间单位：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_fs">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>fs</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ps">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ps</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ns">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
